src/VecSim/algorithms/hnsw/hnswlib.h

```diff
@@ -31,7 +31,7 @@ template <typename dist_t>
 struct CompareByFirst {
```

Comment:
I noticed there is an identical function in vector_block.h. Maybe this should be implemented there as well?

Reply:
Right, I'll add this addition to `vector_block.h` as well, thanks!